ContentMetadata/Resources/Sql/Install.sql

```diff
@@ -101,4 +101,16 @@ END
 GO
 
 GRANT EXECUTE ON [dbo].[custom_Metadata_Delete_Key] TO PUBLIC
+GO
+
+IF EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'[dbo].[custom_Metadata_List_Key]') AND type in (N'P', N'PC'))
```

Comment:
This is where we should also be passing in the value and filtering based on the value, not just returning all entities regardless of their value.